repositories/user_repository.py

```
@@ -0,0 +1,64 @@
+import json
+import os
+from typing import Optional
+
+from common.config import settings
+
+
+class UserRepository:
+    def __init__(self):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)